src/plugins/visualize/kibana.json

```diff
@@ -9,7 +9,7 @@
     "navigation",
     "savedObjects",
     "visualizations",
-    "dashboard"
```

Comment:
Let's not remove dashboard as a dependency just yet.

Reply:
Makes sense, I've restored it and we can remove it in your visualize without saving PR.